main.js

```diff
@@ -51,7 +51,6 @@ const getAppContainer = (options) => {
 		{
 			withBackendAuthentication: true,
 			withFlags: false,
-			withAb: false,
```

Comment:
**suggestion (non-blocking)**: could we add a deprecation notice so that apps still passing in this option get prompted to remove it?

Reply:
Yep, good idea, will add.